dace/sdfg/propagation.py

```diff
@@ -772,3 +772,85 @@ def propagate_memlet(dfg_state,
         new_memlet.num_accesses = 0
 
     return new_memlet
+
+
+def propagate_symbol(dfg_state,
```

Comment:
Is it possible to use the function from #394 instead? If so we should consider merging it, since the refactor there is nice for more uses (state propagation).

Reply:
Done